src/main.cc

```diff
@@ -109,23 +131,36 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
-  // Generate animation.
-  const char* output = argv[2];
+  // Generate the animation.
+  const std::string& output = absl::GetFlag(FLAGS_o);
   WebPData webp_data;
   WebPDataInit(&webp_data);
 
+  // By default, use lossy encoding and impose the same quality to all frames.
+  libwebp::Thumbnailer::Method method =
+      libwebp::Thumbnailer::Method::kEqualQuality;
+
+  if (absl::GetFlag(FLAGS_equal_psnr)) {
+    method = libwebp::Thumbnailer::Method::kEqualPSNR;
+  } else if (absl::GetFlag(FLAGS_near_ll_diff)) {
+    method = libwebp::Thumbnailer::Method::kNearllDiff;
+  } else if (absl::GetFlag(FLAGS_near_ll_equal)) {
+    method = libwebp::Thumbnailer::Method::kNearllEqual;
+  } else if (absl::GetFlag(FLAGS_slope_optim)) {
+    method = libwebp::Thumbnailer::Method::kSlopeOptim;
+  }
```

Comment:
I'm not sure if absl::Flags can handle this, since it would be awkward checking by hand.

Reply:
The four `ABSL_FLAG(bool)` could be merged into a single `ABSL_FLAG(std::string)` so that the last method arg is used. I do not know if this is better though.